app/assets/stylesheets/mixins/vars.sass

```diff
@@ -5,10 +5,11 @@ $raleway-bold: "raleway-bold", sans-serif
 $raleway-medium: "raleway-medium", sans-serif
 $raleway-semi: "raleway-semi", sans-serif
 
-$mobile-width: 320px
-$tablet-width: 640px
+$mobile-width: 480px
+$tablet-width: 768px
 $laptop-width: 992px
 $desktop-width: 1300px
+$large-desktop-width: 1800px

```

Comment:
The responsive breakpoints were intentionally changed from the original design. To match the design of the content rather than actual device sizes.

These represent the smallest values so.
Mobile should be 639px and less
Tablet should be 640px - 991px
Laptop should be 992px - 1299px
Desktop should be 1300px and higher

Reply:
Use the breakpoints currently used on master.

Mobile should be 639px and less
Tablet should be 640px - 991px
Laptop should be 992px - 1299px
Desktop should be 1300px and higher

The designs for 320px and 768px should correspond to Mobile (0px - 639px)
The design for 960px should correspond to Tablet (640px - 991px)
The design for 1366px should correspond to Laptop (992px - 1365px)
The design for 2560px should correspond to Desktop (1366px and higher)

Reply:
I realize it will be awkward to do the laptop one, since the design is larger than the actual breakpoints.